{
  try {
    final XQQueryException ex=new XQQueryException("Hello world!");
    assertEquals("A-XQQEX-1.1: XQQueryException successfully created","Hello world!",ex.getMessage());
  }
 catch (  final Exception e) {
    fail("A-XQQEX-1.1: Creating XQQueryException faild with message: " + e.getMessage());
  }
}
